docs/outlook/outlook-add-ins-overview.md

```diff
@@ -45,6 +45,9 @@ Outlook add-ins are available on messages or appointments while composing or rea
 
 - Protected by Information Rights Management (IRM) or encrypted in other ways for protection. A digitally signed message is an example since digital signing relies on one of these mechanisms.
 
+  > [!IMPORTANT]
+  > Starting with Outlook build 13120.1000 on Windows, add-ins now activate on items protected by IRM. This feature is currently [in preview](../reference/requirement-sets/outlook-api-requirement-sets.md#using-preview-apis).
```

Comment:
Add-ins will activate on IRM items only if the Usage Right to Allow Programmatic Access is enabled (OBJMODEL defined here): https://docs.microsoft.com/en-us/azure/information-protection/configure-usage-rights#usage-rights-and-descriptions